src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -1105,36 +1120,41 @@ private void BuildErrorBubble()
             ErrorBubble.DismissedMessages.CollectionChanged += DismissedNodeMessages_CollectionChanged;
         }
 
+        // These colors are duplicated from the DynamoColorsAndBrushesDictionary as it is not assumed that the xaml will be loaded before setting the color
+        // SharedDictionaryManager.DynamoColorsAndBrushesDictionary["NodeErrorColor"];
+        private static SolidColorBrush errorColor = (SolidColorBrush)(new BrushConverter().ConvertFrom("#EB5555"));
```

Comment:
this is kind of a bummer for maintainability, I'm fine with it for 2.13 - but I'm assuming if the color gets set before the view is loaded it's better to just deal with the null or exception and handle it later when the view is actually loaded. For now can you add a TODO / file a task as tech debt to improve this?